lib/utils/serializers.dart

```diff
@@ -125,6 +128,7 @@ part 'serializers.g.dart';
   PickProfilePic,
   PlatformType,
   PlumbDatabaseStream,
+  PushPage,
   PrintFCMToken,
   RecordAddedRouteInfo,
   RecordRemovedRouteInfo,
```

Comment:
does removeCurrentPage need to be here?

Reply:
Yes indeed!  Great pickup - thanks!  I've added it now.